ci/docker/core.build.Dockerfile

```diff
@@ -11,6 +11,9 @@ SHELL ["/bin/bash", "-ice"]
 
 COPY . .
 
+RUN apt install libmpich-dev
+
```

Comment:
put this before `COPY .` ? if it does not depend on the source code, it should be put before.

also, if this is required for core in general. it probably should go into `oss-ci-base_build`?

Reply:
Hi @aslonnie it's not required for core in general. It's only useful when the user want to run mpi on top of ray. And in this case, they should install it by themselves.